docs/cristin.ttl

```diff
@@ -81,6 +81,10 @@
     """ .
 
 
+:VARBEID_VDISIPLIN_Class
+    a           :CristinClass ;
+    rdfs:comment "UNKNOWN" .
+
```

Comment:
Better to ask Daniel Sachse about the details of these UKNOWN fields and fill in the information